code/script.js

```diff
@@ -0,0 +1,242 @@
+      // in 'json[index].fork' we can find the boolean (true/false)
+      // we want all the repo with boolean true
+
+      // we also want to filter and find the projects from Technigo
+      // the repos from Technigo always start with 'project-'
+      const filteredTechnigoRepos = json.filter(
+        (repo) => repo.fork && repo.name.includes('project-')
+      );
+
+      // store the project names in a new array called namesOfTheRepos
+      const namesOfTheRepos = filteredTechnigoRepos.map((repo) => repo.name);
+
+      // for each project, create a button element and class project-card
+      // inside each card display the project name and a emoji
+      // with append we push each project card
+      // and add an eventListener so we can click on each card and get a modal(popup)
+      namesOfTheRepos.forEach((projectName) => {
+        const projectCard = document.createElement('button');
+        projectCard.classList.add('project-card');
+        projectCard.innerHTML = `${projectName} <div class="project-emoji">👆</div>`;
+        projectContainer.append(projectCard);
+        projectCard.addEventListener('click', () => getModal(projectName));
+      });
+
+      // invoke a function to draw a chart, where we can see finished and unfinished project throughout the boot camp
+      // the function is declared in chart.js
+      drawProgressChart(filteredTechnigoRepos.length);
+    });
+};
+
+// function to get information about the GitHub user
+const getUserInfo = () => {
+  fetch(API_USER)
+    .then((response) => {
+      return response.json();
+    })
+    .then((json) => {
+      // display profile image, user and contact info
+      profile.innerHTML += `
+      <div class="profile-info">
+        <table> 
+          <tr>
+            <td>Username:</td>
+            <td>${json.login}</td>
+          </tr>
+          <tr>
+            <td>Name:</td>
+            <td>${json.name}</td>
+          </tr>        
+          <tr>
+            <td>Location:</td>
+            <td>${json.location}</td>
+          </tr>
+        </table>
+        <p>Bio: ${json.bio}</p>
+      </div>
+      <div class="profile-image-icons">
+        <img class="profile-image" src="${json.avatar_url}" alt="profile image"/>
+        <div class="contact-icons"> 
+          <a href="https://www.linkedin.com/in/katie-wu-213a82150/" target="blank">
+            <img class="icon" src="./assets/linkedin.png" alt="linkedin icon" width="16px" height="16px"/>
+          </a>
+          <a href="https://github.com/katiewu1/" target="blank">
+            <img class="icon" src="./assets/github.png" alt="github icon" width="16px" height="16px"/>
+          </a>
+          <a href="mailto:katiewu.92@gmail.com">
+            <img class="icon" src="./assets/email.png" alt="email icon" width="16px" height="16px"/>
+          </a>
+        </div>
+      </div>
+
+      `;
+    });
+};
+
+// function to get a modal(popup)
+const getModal = (projectName) => {
```

Comment:
Really cool with the extra level, that you can klick on the buttons for more info. Smart to use the loading GIF.
(the pop-up window appears on different hights on my screen though, so I have to scroll down a bit to read.)